# Array of detections from YOLO
std_msgs/Header header
Detection[] detections
int32 frame_width
int32 frame_height
